tests/queries/tests.py

```diff
@@ -2104,6 +2104,13 @@ def test_sliced_delete(self):
         DumbCategory.objects.filter(id__in=DumbCategory.objects.order_by('-id')[1:]).delete()
         self.assertEqual(set(DumbCategory.objects.values_list('id', flat=True)), {3})
 
+    def test_subquery_different_database(self):
```

Comment:
How about putting the test in `tests/lookup/tests.py` (near `test_in` I was thinking)?
Please squash your commits.